scipy/stats/_continuous_distns.py

```diff
@@ -6499,7 +6502,7 @@ class argus_gen(rv_continuous):
         f(x, \chi) = \frac{\chi^3}{\sqrt{2\pi} \Psi(\chi)} x \sqrt{1-x^2}
                      \exp(- 0.5 \chi^2 (1 - x^2))
 
-        where:
+    for 0 < x < 1, where
```

Comment:
Seems right with c = 1 according to https://en.wikipedia.org/wiki/ARGUS_distribution.
Although while you're at it you might change 0.5 to \frac{1}{2}

Reply:
Yeah, that `0.5` does look out of place.  Instead of using `\frac{1}{2}`, I went with `\exp(-\chi^2 (1 - x^2)/2)`.

I have a love/hate thing with LaTeX in docstrings.  I love the rendered output on the web page, but I hate having to read raw LaTeX in docstrings when I'm working in a shell, so I'd rather keep the LaTeX simple when possible.

(Is anyone out there working on providing LaTeX/Sphinx processing of docstrings in an ipython or jupyter shell?  Does it already exist?)

Reply:
I know some editors / configurations can do this (e.g., Spyder, Jupyter Notebook) but don't know of any doing it in the terminal itself. Apparently this [is possible in principle](https://unix.stackexchange.com/questions/353033/latex-expressions-shown-in-a-terminal), though.

Reply:
> I know some editors / configurations can do this (e.g., Spyder, Jupyter Notebook) 

Can they render a docstring in the notebook (without just linking to the web page)?  I.e. if I import `argus` from `scipy.stats`, and do `argus?` in a jupyter notebook, can it be configured to render that docstring (using sphinx, I guess) on the fly?  By default, it just shows the plain text.

Reply:
Sorry, I thought I had read this was possible with Jupyter but looking deeper it's not.

Spyder does work, though, if you want to switch to that IDE.